kernel/src/hil/acifc.rs

```diff
@@ -0,0 +1,40 @@
+//! Interface for direct control of ACIFC
+//!
+//! Author: Danilo Verhaert <verhaert@cs.stanford.edu>
+
+use returncode::ReturnCode;
+
+/// Enum for selecting which edge to trigger interrupts on.
+// #[derive(Debug)]
+// pub enum InterruptMode {
+//     RisingEdge,
+//     FallingEdge,
+//     EitherEdge,
+// }
+
+pub trait Acifc {
+    /// Initialize the ACIFC
+    fn initialize_acifc(&self) -> ReturnCode;
```

Comment:
Is there a reason this needs to happen explicitly? Is there _always_ an initialization step for analog comparator hardware? If so, what is it meant to do? What are the expectations the caller can have after calling this method?

Reply:
Two steps are always necessary: Enabling the clock, and enabling the ACIFC by writing a 1 to the Enable register. Do you agree that calling an initialization function is better than always calling the clock and writing to this register?